torchensemble/tests/test_logging.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from torchensemble.utils.logging import set_logger
+
+
+def record(logger):
+    logger.debug("Debug!")
+    logger.info("Info!")
+    logger.warning("Warning!")
+    logger.error("Error!")
+    logger.critical("Critical!")
+
+
+def test_logger():
+    logger = set_logger("Loglevel_DEBUG", "DEBUG")
+    record(logger)
+    logger = set_logger("Loglevel_INFO", "INFO")
+    record(logger)
+    logger = set_logger("Loglevel_WARNING", "WARNING")
+    record(logger)
+    logger = set_logger("Loglevel_ERROR", "ERROR")
+    record(logger)
+    logger = set_logger("Loglevel_CRITICAL", "CRITICAL")
+    record(logger)
+
+    with pytest.raises(ValueError):
+        set_logger("Loglevel_INVALID", "INVALID")
```

Comment:
How about adding another line that makes sure that the error message is exactly what we expected. Something like https://github.com/xuyxu/Ensemble-Pytorch/blob/master/torchensemble/tests/test_set_optimizer.py#L44